shared/scss/base/_fonts.scss

```diff
@@ -36,6 +36,14 @@
 @font-face {
     font-family: 'DDG_ProximaNova';
     src: url('../font/ProximaNova-Sbold-webfont.woff') format('woff');
-    font-weight: 600;
-    font-style: bold;
+    font-weight: 700;
```

Comment:
This doesn't actually change anything. I looked through the extension code and all of the places that we use bold text, we just user `font-weight: bold`, which actually means `font-weight: 700`. Also, `bold` is not a valid font-style according to https://www.w3schools.com/cssref/pr_font_font-style.asp.

I had to make this change in order to add and use the 800 weight font below, without it `font-weight: bold` was translating into `font-weight: 700`, which then actually renders `font-weight: 800` since there was no font with weight 700.